src/command/mcl.zig

```diff
@@ -1896,6 +1959,186 @@ fn mclWaitRecoverSlider(params: [][]const u8) !void {
     }
 }
 
+/// Create a logging file for logging register values specified by
+/// the desired register and desired axis. Every successfull setup
+/// can be accessed through log_line variable.
+fn setLogRegisters(params: [][]const u8) !void {
+    const line_name: []const u8 = params[0];
+    const line_idx: usize = try matchLine(line_names, line_name);
+    const line: mcl.Line = mcl.lines[line_idx];
+
+    var log: RegisterLogging = undefined;
+    log.line_index = @intCast(line_idx);
+
+    // Iterate over axis list from parameters
+    var station_indices: []u8 = try allocator.alloc(u8, line.stations.len);
```

Comment:
This needs an `errdefer` to free, and typing should be using `mcl.Station.Index`